racetime/views/user.py

```diff
@@ -134,6 +134,27 @@ def get(self, request, *args, **kwargs):
         return resp
 
 
+class UserProfileData(ViewProfile):
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        user = self.object.api_dict_summary()
+        del user['can_moderate']
```

Comment:
> I had to specifically exclude can_moderate as under a user context it would not be relevant.

This is okay, the API documentation makes it clear that `can_moderate` would be `False` in this context. See https://github.com/racetimeGG/racetime-app/wiki/Public-API-endpoints#user-data

> `can_moderate`: A boolean indicating if the user is a moderator. This is context-sensitive, as moderator status varies per category. If there is no category in context, this will always be false (even if user is staff).

I think `can_moderate` should be left in for consistency with other API endpoints that return user data.

Reply:
I rolled with it as requested. But as is adding in the stats [] block makes that moot. It won't be a standard data set at that point along with the rest of them.